lib/tasks/content_tagged_to_ancestors.rake

```diff
@@ -0,0 +1,11 @@
+namespace :content do
+  desc "Find all content that is tagged to a taxon and its direct ancestor and removes the redundant tagging"
+  task content_tagged_to_ancestor: :environment do
```

Comment:
This task is in the `content` namespace, so you could probably lose that from the front of the task name, so it would be `content:tagged_to_ancestor` rather than `content:content_tagged_to_ancestor`.